### PR TITLE
worlds update

### PR DESCRIPTION
**Pull request by antonofka**
Updated world files for simulation and EX

### CODE REVIEW

File: worlds/C1/1.wbt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../ICTLab.proto"
+EXTERNPROTO "../PipeColor.proto"
```
Comment (GertKanter) [inline worlds/C1/1.wbt:7]: This file is not included in the PR, please add

File: worlds/C3/1.wbt [hunk 2]
```diff
@@ -0,0 +1,78 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../ICTLab.proto"
+EXTERNPROTO "../LabMazeWall.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/C3/1.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../PipeColor.proto")

File: worlds/C3/1.wbt [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/1.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/1.wbt [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/1.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/2.wbt [hunk 5]
```diff
@@ -0,0 +1,89 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../ICTLab.proto"
+EXTERNPROTO "../LabMazeWall.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/C3/2.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/2.wbt [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/2.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/2.wbt [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/2.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/3.wbt [hunk 8]
```diff
@@ -0,0 +1,70 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../ICTLab.proto"
+EXTERNPROTO "../LabMazeWall.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/C3/3.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/3.wbt [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/3.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/3.wbt [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/3.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/4.wbt [hunk 11]
```diff
@@ -0,0 +1,84 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../ICTLab.proto"
+EXTERNPROTO "../LabMazeWall.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/C3/4.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/4.wbt [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/4.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/4.wbt [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/4.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/5.wbt [hunk 14]
```diff
@@ -0,0 +1,89 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../ICTLab.proto"
+EXTERNPROTO "../LabMazeWall.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/C3/5.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/5.wbt [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/5.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/5.wbt [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/5.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/6.wbt [hunk 17]
```diff
@@ -0,0 +1,113 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../ICTLab.proto"
+EXTERNPROTO "../LabMazeWall.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/C3/6.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/6.wbt [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/6.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/C3/6.wbt [hunk 19]
```diff
@@ -0,0 +1,113 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/C3/6.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/3/3.wbt [hunk 20]
```diff
@@ -0,0 +1,100 @@
...
+  hidden rotation_7 0 0 1 0.015999305791290327
+  hidden linearVelocity_7 7.245973890543127e-09 7.499010552753238e-09 3.2041755787098974e-09
+  hidden angularVelocity_7 2.581342020882176e-09 1.3601305423986234e-07 -4.450990305787877e-10
+  hidden translation_8 0 -0.1165 0.04021429228087137
+  hidden position_8_0 0.015999307717871517
+  hidden linearVelocity_8 1.5427913975518614e-09 -3.8646289303684475e-09 3.252586844304517e-09
+  hidden angularVelocity_8 1.8064413933361549e-09 1.3586236146681975e-07 3.8884075465709127e-10
+  hidden rotation_9 0 0 1 0.015999307717881096
+  hidden linearVelocity_9 7.01898729398254e-09 -7.716808578082034e-09 3.188426930186967e-09
+  hidden angularVelocity_9 1.6739459532872767e-09 1.360267646184355e-07 3.8886403306775166e-10
```
Comment (GertKanter) [inline worlds/EX/3/3.wbt]: I think the world has been simulated before saving the world?

File: worlds/EX/6/1.wbt [hunk 21]
```diff
@@ -0,0 +1,50 @@
+#VRML_SIM R2023b utf8
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/EX/6/1.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/6/2.wbt [hunk 22]
```diff
@@ -0,0 +1,59 @@
+#VRML_SIM R2023b utf8
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/EX/6/2.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/6/3.wbt [hunk 23]
```diff
@@ -0,0 +1,61 @@
+#VRML_SIM R2023b utf8
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/EX/6/3.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/7/1.wbt [hunk 24]
```diff
@@ -0,0 +1,73 @@
...
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
+EXTERNPROTO "../../LabMazeWall.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
```
Comment (GertKanter) [inline worlds/EX/7/1.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/7/1.wbt [hunk 25]
```diff
@@ -0,0 +1,73 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/EX/7/1.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/7/1.wbt [hunk 26]
```diff
@@ -0,0 +1,73 @@
...
+EXTERNPROTO "../../LabMazeWall.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/EX/7/1.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/7/2.wbt [hunk 27]
```diff
@@ -0,0 +1,91 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/EX/7/2.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/7/2.wbt [hunk 28]
```diff
@@ -0,0 +1,91 @@
...
+EXTERNPROTO "../../LabMazeWall.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/EX/7/2.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/7/3.wbt [hunk 29]
```diff
@@ -0,0 +1,72 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../ICTLab.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/PipeColor.proto"
```
Comment (GertKanter) [inline worlds/EX/7/3.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

File: worlds/EX/7/3.wbt [hunk 30]
```diff
@@ -0,0 +1,72 @@
...
+EXTERNPROTO "../../LabMazeWall.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Yellow.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Red.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Blue.proto"
+
+
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/ConeShape.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/CubeShape.proto"
```
Comment (GertKanter) [inline worlds/EX/7/3.wbt]: Adding a proto like this means that the proto itself is not allowed to include nested EXTERNPROTOS (must be added as "../*****.proto")

**Review by GertKanter — CHANGES_REQUESTED** [review]
Please fix the issues!

**Comment by antonofka** [comment]
updated protos location and fixed the issues

**Review by antonofka — COMMENTED** [review]
changes have been made to new commit fixing the current issues

File: worlds/EX/3/3.wbt [hunk 31]
```diff
@@ -0,0 +1,72 @@
+#VRML_SIM R2023b utf8
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/objects/backgrounds/protos/TexturedBackground.proto"
+EXTERNPROTO "https://raw.githubusercontent.com/taltech-roboprog/protos/refs/heads/main/Turtlebot4.proto"
+EXTERNPROTO "../../Pipe.proto"
+EXTERNPROTO "../..LabMazeWall.proto"
```
Comment (GertKanter) [inline worlds/EX/3/3.wbt]: typo

**Review by GertKanter — CHANGES_REQUESTED** [review]

File: worlds/EX/3/3.wbt (hunk 31 above)
Comment (antonofka) [reply worlds/EX/3/3.wbt]: fixed in new commit

**Review by GertKanter — APPROVED** [review]

**Pull request merged**